src/Microsoft.Extensions.OData.Migration/Microsoft.Extensions.OData.Migration.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Version>0.9.2-preview</Version>
+    <Authors>Microsoft</Authors>
```

Comment:
These setting maybe not work with "<import project="..\Build.props" />, Just FYI.